stylus/components/avatar.styl

```diff
@@ -3,6 +3,8 @@
 $avatar
     background-color var(--paleGrey)
     color var(--silver)
+    position relative
+    z-index 1
```

Comment:
we do have a z-index.styl with css var. We should use for instance `$low-index` instead